browser/mapping.go

```diff
@@ -249,7 +249,7 @@ func mapElementHandle(vu moduleVU, eh *common.ElementHandle) mapping {
 		},
 	}
 	maps["$"] = func(selector string) (mapping, error) {
-		eh, err := eh.Query(selector)
+		eh, err := eh.Query(selector, false)
```

Comment:
WDYT about putting `false` in a package-level constant like `strictModeOff` to be more explicit?

Reply:
Resolved in d7f3f532c35291924fc0f78f1c9985a7a61d5565